cpp/src/rolling/range_window_bounds.cpp

```diff
@@ -35,7 +35,8 @@ struct range_scalar_constructor {
   {
     CUDF_FAIL(
       "Unsupported range type. "
-      "Only Durations, fixed-point, and non-boolean integral range types are allowed.");
+      "Only Durations, fixed-point, floating point, and non-boolean integral range types are "
+      "allowed.");
```

Comment:
```suggestion
      "Only Durations, fixed-point, and non-boolean numeric range types are allowed.");
```
We could perhaps use `numeric` here as well.

Reply:
Fixed here as well.